js/main.js

```diff
@@ -0,0 +1,59 @@
+const getRandomInteger = function (num1, num2) {
```

Comment:
Про единообразие нейминга [имелось в виду](https://github.com/htmlacademy-javascript/1555297-keksobooking-22/pull/2#discussion_r567249177) это:
```suggestion
const getRandomRoundedNumber = function (num1, num2) {
```
Так нейминг становится единообразным с `getRandomFloatNumber`

Reply:
Также стоит добавлять строгий режим во все js файлы - [критерий Б16](https://up.htmlacademy.ru/javascript/22/criteries#b16)